pyproject.toml

```
@@ -31,21 +31,21 @@ exclude = [ "mac_cleanup/__main__.py", "mac_cleanup/py.typed" ]
 
 
 [tool.poetry.dependencies]
-python = ">=3.10,<3.14"
+python = ">=3.10,<3.15"
 rich = "^13.9.4"
 attrs = "~25.3.0"
 inquirer = "^3.4.0"
 toml = "~0.10.2"
-beartype = "~0.20.0"
+beartype = "~0.22.0rc0"
```

```suggestion
beartype = "~0.22.1"
```
https://pypi.org/project/beartype